### PR TITLE
added log to set

### PR DESCRIPTION
**Pull request by skailer52**
Added a log to the project

### CODE REVIEW

File: input_history.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ class FifoCache {
         else {
             this.cache.push(value);
         }
+        console.info("Value has been set")
```
Comment (asadm-skael) [inline input_history.js:40]: please use console.log instead of console.info()

**Review by asadm-skael — CHANGES_REQUESTED** [review]

File: input_history.js (hunk 1 above)
Comment (skailer52) [reply input_history.js:40]: Ok will make a pr soon

Comment (skailer52) [reply input_history.js]: Ok this change was added with 1f8cec2

**Review by asadm-skael — APPROVED** [review]
changes look good !

**Pull request merged**